dev/io.openliberty.io.smallrye.config/src/io/openliberty/microprofile/config/internal/extension/OLSmallRyeConfigBuilder.java

```diff
@@ -29,6 +29,14 @@
 
     private static final TraceComponent tc = Tr.register(OLSmallRyeConfigBuilder.class);
 
+    public OLSmallRyeConfigBuilder() {
+        super();
+        //I have no idea why this property would default to true in the SmallRye code
+        //With it set to true, you get a validation error on hundreds of properties
+        //At the moment, I don't really understand what it is for
+        withValidateUnknown(false);
```

Comment:
Should be able to remove this line again if this issue is resolved in a future version:
https://github.com/smallrye/smallrye-config/issues/548